### DISCUSSION TITLE
Translations

### DISCUSSION THREAD

**Discussion by nimbit-software** [gorules/jdm-editor#114] [Ideas]
How would you guys feel if i started adding Translation (i18n) to the editor with a language switcher. Making it possible for external translation files to be added as well? Atm i dont think there are thaaaat many texts so it would still be manageable to add it all. 

If its something you guys are interested in a can get that done and create a PR. 

Assuming u do want it let me know if you have a preferance for a i18n library.

**Comment by ivanmiletic** [comment]
Hi, we would be happy if you would like to add localisation concept - i would suggest just to wait a week or two as we are adding diff view capability in editor.

**Reply by d-sooter** [reply]
Sure thing you can get back to me and we can pick an i18n lib and ill start adding it in

**Reply by ivanmiletic** [reply]
This is the one we are waiting: https://github.com/gorules/jdm-editor/pull/118

Ill get back to you in a couple of days

**Reply by ivanmiletic** [reply]
@d-sooter please check this package json i have fixed it a bit: [packages/jdm-editor/package.json](https://github.com/gorules/jdm-editor/pull/118/files)

**Reply by d-sooter** [reply]
@ivanmiletic Also is it possible to set draggable and deleteable to false (react flow props)
